Libraries/Components/Picker/PickerWindows.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Can we put this whole file in:
./ReactWindows/js/Components/Picker/PickerWindows.js?

We're trying to keep all the Windows files isolated for now.